res/country_metadata/orchardProduces.yml

```diff
@@ -1,214 +1,205 @@
+CK: [coconut, papaya, guava, mango, mangosteen, tomato, pineapple, avocado, banana, orange, coffee]
+CL: [grape, walnut, cherry, apple, avocado, hazelnut, olive, cranberry, plum, peach, tomato, almond, kiwi, pear, lemon, lime]
+CM: [cacao, coffee, kola_nut, tomato, banana, rubber, avocado, chilli_pepper, sweet_pepper, pineapple, tea, coconut, pepper, apricot, date, peach]
+CN: [tea, orange, apple, plum, tomato, persimmon, pear, peach, grape, rubber, banana, chestnut, walnut, kiwi, guava, mango, mangosteen]
+CO: [coffee, cacao, banana, avocado, guava, mango, mangosteen, lemon, lime, pineapple, coconut, orange, agave, chilli_pepper, sweet_pepper, tomato, rubber, papaya, grape]
+CR: [coffee, banana, pineapple, orange, cacao, guava, mango, mangosteen, avocado, grapefruit, lemon, lime, coconut, tomato, papaya, pepper, chilli_pepper, sweet_pepper, strawberry]
+CU: [guava, mango, mangosteen, banana, tomato, coffee, papaya, agave, coconut, chilli_pepper, sweet_pepper, pineapple, grapefruit, orange, cacao, grape, sisal, lemon, lime]
+CV: [coconut, tomato, banana, coffee, chilli_pepper, sweet_pepper, guava, mango, mangosteen]
+CY: [olive, grape, almond, orange, lemon, lime, grapefruit, plum, apple, peach, tomato, cherry, walnut, banana, apricot, fig]
+CZ: [grape, apple, hop, plum, cherry, apricot, pear, strawberry, peach, chilli_pepper, sweet_pepper, tomato, walnut, raspberry]
+DE: [grape, apple, hop, strawberry, plum, blueberry, pear, cherry, raspberry, tomato, walnut, apricot, peach, chilli_pepper, sweet_pepper]
+DJ: [chilli_pepper, sweet_pepper]
+DK: [apple, strawberry, pear, cherry, blueberry, plum, raspberry, tomato]
+DM: [cacao, banana, coconut, guava, mango, mangosteen, grapefruit, orange, coffee, lemon, lime, avocado, tomato, pineapple, nutmeg, papaya]
+DO: [cacao, coffee, coconut, avocado, banana, lemon, lime, orange, papaya, cashew_nut, pineapple, chilli_pepper, sweet_pepper, tomato, guava, mango, mangosteen, grapefruit, sisal]
+DZ: [olive, date, grape, fig, almond, apple, apricot, tomato, chilli_pepper, sweet_pepper, pear, peach, orange, plum, lemon, lime, cherry]
+EC: [cacao, banana, coffee, orange, guava, mango, mangosteen, rubber, lemon, lime, avocado, pineapple, papaya, agave, pear, coconut, chilli_pepper, sweet_pepper, pepper]
+EE: [strawberry, apple, raspberry, plum, tomato]
+EG: [tomato, guava, mango, mangosteen, orange, olive, grape, date, banana, apple, fig, chilli_pepper, sweet_pepper, peach, strawberry, lemon, lime, apricot, walnut]
+ES: [olive, grape, almond, orange, peach, tomato, lemon, lime, chestnut, apple, chilli_pepper, sweet_pepper, pear, apricot, fig, avocado, plum]
+# Canary Islands has very different climates, biomes and culture (manually edited)
+IC: [olive, grape, almond, orange, peach, tomato, lemon, lime, chestnut, apple, chilli_pepper, sweet_pepper, pear, apricot, fig, avocado, plum, pineapple, mango, avocado]
```

Comment:
BTW @erik55 note as I only saw your https://github.com/streetcomplete/StreetComplete/pull/2596 now, note you may also list your added items at the beginning if you think they are more often grown there in contrast to the other ones.

One could e.g. reorder it like this (of course I have no idea whether that is somewhat realistic or so, it's just a guess):

```suggestion
# Canary Islands has very different climates, biomes and culture (manually edited)
IC: [fig, avocado, plum, pineapple, mango, avocado, apricot, olive, grape, almond, orange, peach, tomato, lemon, lime, chestnut, apple, chilli_pepper, sweet_pepper, pear]
```

For now, I just re-added the ones you added at the end, as you did, to keep the current way as it is.